docs/modules/ROOT/pages/runbooks/unusedcapacity.adoc

```diff
@@ -0,0 +1,50 @@
+= Alert Group: syn-UnusedCapacity
+
+We provide a rule to notice if a cluster has unused nodes.
+The cost of running the cluster may be optimized by removing unused nodes.
+
+== Alert Rule: SYN_ClusterHasUnusedNodes [[SYN_ClusterHasUnusedNodes]]
+
+=== icon:glasses[] Overview
+
+This alert indicated that the cluster is underutilized and nodes might be removed.
+It will fire if *pod count*, *CPU requests*, *CPU usage*, *memory requests* and *memory usage* are lower than the threshold of reserved nodes used for failover and future capacity needs.
```

Comment:
```suggestion
The alert fires if *pod count*, *CPU requests*, *CPU usage*, *memory requests*, and *memory usage* are are all low enough that a node can be removed without impacting cluster stability in case of a node failure.
By default, the alert will fire when the equivalent of at least four nodes has been unused for each metric for eight hours.
Both the threshold of four unused nodes, and the duration of eight hours can be tuned through the component parameters.
```